x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/stats/SearchStats.java

```diff
@@ -161,14 +161,16 @@ public boolean isSingleValue(String field) {
             if (exists(field) == false) {
                 stat.singleValue = true;
             } else {
-                var sv = new boolean[] { false };
+                var sv = new boolean[] { true };
```

Comment:
Was this a bug...?

Reply:
I guess it's safe to assume that there will be at least one valid MappedFieldType, and if the type allows MV it will be set to `false` anyway.
Or maybe there are situations where none of the contexts has a MappedFieldType for that specific field, and in that case we still want to fall back to `false`?